docs/notebooks/BoostHistogramHandsOn.ipynb

```diff
@@ -1 +0,0 @@
-../../notebooks/BoostHistogramHandsOn.ipynb
```

Comment:
Have no idea about this :/

Reply:
Whitespace issue?

Reply:
Different file permissions? Odd, not sure I've seen this show up this way before...

Reply:
I bet if we squash your history, this will go away.

Reply:
Alright. See how it will be after the squash.

Reply:
I can do a squash and force-push, is that okay?